docs/MigrationGuide.md

```diff
@@ -26,6 +26,9 @@ To see the biggest differences please consult the [changelog](Changelog.md).
 ### Other changes
 
 * [#5574](https://github.com/TouK/nussknacker/pull/5574) Removed the support for the plugable expression languages: `ExpressionConfig.languages` removed
+* [#5724](https://github.com/TouK/nussknacker/pull/5724) Improvements: Run Designer locally
+  * Introduce `JAVA_DEBUG_PORT` to run the Designer locally with remote debugging capability
+  * Removed `NUSSKNACKER_SCALA_VERSION` from running Designer locally, please use `SCALA_VERSION` instead
```

Comment:
Why did you change it from `NUSSKNACKER_SCALA_VERSION` to `SCALA_VERSION`? Thanks to nussknacker prefix, it was possible to set up this on the system level (in `.profile` file). I used it quite often when for some reasons I had to switch to 2.12 for a longer period.

Reply:
Hmm, ok it has sense.. But right now we use SCALA_VERSION on run scripts like ./runServer / CI scripts and NUSSKNACKER_SACLA_VERSION on sbt.. IMO, we should normalize it and use one convention everywhere. We can stay with NUSSKNACKER_SCALA_VERSION, WDYT?